### ISSUE TITLE
Add UC Mode customization options

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#2186]
## Add UC Mode customization options

Since the default disconnect/reconnect time might not be enough for some websites with certain internet connection speeds, there should be a way to customize that.

(Note: The UC Mode disconnect/reconnect time is the time that chromedriver disconnects from Chrome before reconnecting again. When chromedriver is connected to Chrome, websites may be able to detect it, and that's why it should be disconnected temporarily from Chrome when opening a website. After a moment, it should be safe to reconnect again without triggering bot-detection services.)

A `reconnect_time` arg will be added to `driver.uc_open_with_reconnect()` so that the reconnect time can be customized.

```python
# Example:
driver.uc_open_with_reconnect("https://nowsecure.nl/#relax", reconnect_time=5)

# Short form example:
driver.uc_open_with_reconnect("https://nowsecure.nl/#relax", 5)
```

The default `reconnect_time` will be set to `2.15` seconds. (This may change. The value will be stored in `seleniumbase.fixtures.constants.UC.RECONNECT_TIME`.)

(Note: `uc_open_with_reconnect()` is part of the `driver`. If using one of the `BaseCase` formats, it'll be accessible via `self.driver.uc_open_with_reconnect()`. For the `pytest` fixture or `SB()` manager, use `sb.driver.uc_open_with_reconnect()`.)

**Comment by MacMarde** [comment]
@mdmintz 
Does this mean that UC can basically not be undetected?

I mean the above solution would not work as soon as anti-bot detection is checking for selenium periodically instead of only once.

**Comment by mdmintz** [comment]
@MacMarde After chromedriver reconnects to Chrome, a webpage could detect chromedriver if checking periodically.

**Comment by MacMarde** [comment]
@mdmintz 
Hmm I think at the moment no one has a reliable solution for selenium.
I am already thinking about converting my project to puppeteer, but that would be a lot of work.

By the way have you read my message [here](https://github.com/seleniumbase/SeleniumBase/issues/2062#issuecomment-1760095783).

**Comment by mdmintz** [comment]
This was resolved in `4.20.2` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.20.2
